.bazelrc

```diff
@@ -136,6 +136,12 @@ build:release-linux           --copt=-march=sandybridge
 build:release-sanitized-linux --copt=-march=sandybridge
 
 build:release-mac --config=release-common --platforms=@//tools/platforms:darwin_x86_64
+build:release-mac-arm64 --platforms=@//tools/platforms:darwin_arm64
+build:release-mac-arm64 --define release=true
+build:release-mac-arm64 --compilation_mode=opt
+build:release-mac-arm64 --config=backtracesymbols
+build:release-mac-arm64 --config=shared-libs
+build:release-mac-arm64 --config=versioned
```

Comment:
Ugh, ok, we can't use `--config=release-common` here because of the `shared-libs` vs. `static-libs` problem.

Reply:
Yes, indeed. We ended up going with making the ARM64 build the exception, rather than changing `release-common` and having to spread the `static-libs` config everywhere else.